backend/src/apiserver/server/util.go

```diff
@@ -241,3 +241,27 @@ func ValidatePipelineSpec(resourceManager *resource.ResourceManager, spec *api.P
 	}
 	return nil
 }
+
+// Verify pipeline version in resource references as creator.
+func VerifyPipelineVersionReferenceAsCreator(resourceManager *resource.ResourceManager, references []*api.ResourceReference) (*string, error) {
```

Comment:
the name of this function is not very clear to me. 
consider CheckPipelineVersionReference()
and put more description what is checked
- pipeline version reference exist
- it is a owner
- ID is a valid 
- pipeline

Reply:
One question: since experiment is the owner of run, I set pipeline version as the creator. Let me know if you prefer it be an owner too.

Reply:
sorry i mean it's a creator instead of a owner.